app/views/categories/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<header class="header-cat">
+<nav class="category-nav">
+<i class="fas fa-bars"></i>
+<h1>Categories</h1>
+<i class="fas fa-search"></i>
+</nav>
+</header>
+<section class="category-container">
+<% if @categories.empty? %>
+<p>There are no categories for the budget</p>
+<% else %>
+<% @categories.each do |category| %>
+    <article class="category-article">
+   
+    <%= link_to user_payment_path(current_user.id, category.id) do %>
+      <div class="article-inner">
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=to%20that%20category.-,When%20the%20user%20clicks%20(or%20taps)%20on%20a%20category%20item%2C%20the%20application%20navigates%20to%20the%20transactions%20page%20for%20that%20category.,-There%20is%20a), When you click (or tap) on a category item, the application should navigate to the transactions page for that category, That doesn't happen it raised an error instead. Please fix that.

![image](https://github.com/cosywasswa/Budget-Buddy/assets/68796232/6b624c0c-0586-4dc4-9052-ada4bae72500)